.github/workflows/build.yml

```diff
@@ -1,6 +1,14 @@
 name: Build cht-core and test against node versions
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
```

Comment:
This is a filter flag. Meaning without pushing to master or creating PR I get no build. Which I'm not always in the position I want to do either of these things but I want to run a build. 

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags